css/style.css

```diff
@@ -0,0 +1,36 @@
+body {
+  background-color: #fcfcff;
+  margin-left: 20%;
+  margin-right: 20%;
+}
+
+#image {
+  width: 90%;
+  height: auto;
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+  max-width: 100%;
+}
+
+#title {
+  text-align: center;
+  margin: 30px;
+  font-size: 60px;
+  color: #161925;
+}
+
+#img-caption {
+  font-size: 20px;
+  text-align: center;
+}
+
+h3 {
+  text-align: center;
+  font-size: 25px;
+}
+
+li {
+  font-size: 20px;
+  margin-bottom: 1rem;
+}
```

Comment:
Please add empty space after each rule. It would make it easier for your teammates to read.